auth_backends/strategies.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from social.strategies.django_strategy import DjangoStrategy
+
+
+class CurrentSiteDjangoStrategy(DjangoStrategy):
```

Comment:
@douglashall thinking about dependencies...the `SOCIAL_AUTH_STRATEGY` can accept any Python path. That means the strategy doesn't necessarily need to live in this repo if we are concerned about dependency injection/contamination/etc. The strategy could live in edx/ecommerce, and you need only update `EdXOpenIdConnect` to use the new methods.